rust/stackablectl/Cargo.toml

```diff
@@ -29,5 +29,9 @@ serde.workspace = true
 snafu.workspace = true
 tera.workspace = true
 tokio.workspace = true
+tokio.features = ["io-std"]
 tracing-subscriber.workspace = true
 tracing.workspace = true
+futures.workspace = true
+termion = "3.0.0"
+libc = "0.2.153"
```

Comment:
nitpick: For now we only need these dependencies for `stackablectl`, but does it maybe make sense to relocate these deps into the root `Cargo.toml` file?

Reply:
Good catch. I tend to mostly let `cargo add` do its thing, and then have a bad habit of not cleaning it up afterwards.

Reply:
3fa1b7ee84a851a54c4b2bb4a5281ee52606cf0a